src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/help.jelly

```diff
@@ -54,6 +54,77 @@ THE SOFTWARE.
     </p>
     <pre>stage name: 'Build', concurrency: 1</pre>
     <p>
-        There are also some predefined variables, shown at the bottom.
+        Many steps require complex nested configuration for a parameter.
+        (And some nested configuration objects in turn have object-typed parameters.)
+        There are three ways these nested objects may be specified, in decreasing order of preference:
+    </p>
+    <ol>
+        <li>
+            <p>
+                Some configuration objects define custom <em>symbols</em>.
+                These are used with a notation looking like a step call, or other function call taking a map of named parameters:
+            </p>
+            <pre>step archiveArtifacts('**.txt')</pre>
```

Comment:
This doesn't actually work like you wrote. at the end of `archiveArtifacts` evaluation, it always gets executed with a meta-step. The evaluation of that method doesn't know that there's a surrounding `step` function.

Is this indicative of a problem in the current code? I understand why people would expect the behaviour you wrote.